modules/29_CES_parameters/calibrate/datainput.gms

```diff
@@ -44,6 +44,7 @@ ppf_29(all_in)
 *** FIXME should include set of "stationary" module, too
   = ppfen_dyn35(all_in)
   + cal_ppf_buildings_dyn36(all_in)
+  + cal_ppf_industry_dyn37(all_in)
   + industry_ue_calibration_target_dyn37(all_in)
```

Comment:
couldn't cal_ppf_industry_dyn37(all_in) take the values of industry_ue_calibration_target_dyn37(all_in) as well?

Reply:
The usage is quite different – so no.
```grep
$ gack cal_ppf_industry_dyn37
modules/29_CES_parameters/calibrate/datainput.gms
47:  + cal_ppf_industry_dyn37(all_in)

modules/37_industry/fixed_shares/sets.gms
71:  cal_ppf_industry_dyn37(all_in)   "primary production factors for calibration - industry"

modules/37_industry/subsectors/sets.gms
211:  cal_ppf_industry_dyn37(all_in)   "primary production factors for calibration - industry"
```

```
$ gack industry_ue_calibration_target_dyn37
modules/01_macro/singleSectorGr/bounds.gms
17:                               OR industry_ue_calibration_target_dyn37(in) )

modules/29_CES_parameters/calibrate/datainput.gms
48:  + industry_ue_calibration_target_dyn37(all_in)
57:  + industry_ue_calibration_target_dyn37(all_in)   !! Industry
244:                              NOT industry_ue_calibration_target_dyn37(all_in) )

modules/29_CES_parameters/calibrate/preloop.gms
65:  pm_cesdata(t,regi,industry_ue_calibration_target_dyn37(in),"price")$(
376:loop (in$(    industry_ue_calibration_target_dyn37(in) 
1107:                                            industry_ue_calibration_target_dyn37(out) 
1118:                          AND NOT industry_ue_calibration_target_dyn37(out)

modules/37_industry/fixed_shares/sets.gms
103:  industry_ue_calibration_target_dyn37(all_in)   "target values of industry calibration"

modules/37_industry/subsectors/bounds.gms
49:                      NOT (t0(t) OR industry_ue_calibration_target_dyn37(in)) ),

modules/37_industry/subsectors/equations.gms
9:q37_energy_limits(ttot,regi,industry_ue_calibration_target_dyn37(out))$( 

modules/37_industry/subsectors/sets.gms
214:  industry_ue_calibration_target_dyn37(all_in)   "target values of industry calibration"
```